tool/net/redbean.c

```diff
@@ -2378,7 +2422,10 @@ static char *HandleMessage(void) {
            kHttpMethod[msg.method], request.path.n, request.path.p,
            msg.headers[kHttpReferer].b - msg.headers[kHttpReferer].a,
            inbuf.p + msg.headers[kHttpReferer].a);
-  if ((a = LocateAsset(request.path.p, request.path.n))) {
+
+  if(HasLuaHandler()) {
```

Comment:
[no action required] This is on the critical path. If there ends up being a measurable impact on latency, we'll likely want to precompute it to a global after .init / .reload.